db/c.cc

```diff
@@ -3862,6 +3887,26 @@ char* rocksdb_options_statistics_get_string(rocksdb_options_t* opt) {
   return nullptr;
 }
 
+uint64_t rocksdb_options_statistics_get_ticker_count(rocksdb_options_t* opt,
+                                                     uint32_t type) {
+  ROCKSDB_NAMESPACE::Statistics* statistics = opt->rep.statistics.get();
+  if (statistics) {
+    return statistics->getTickerCount(type);
+  }
+  return 0;
+}
+
+void rocksdb_options_statistics_get_histogram_data(
+    rocksdb_options_t* opt, uint32_t type,
+    rocksdb_statistics_histogram_data_t* data) {
+  ROCKSDB_NAMESPACE::Statistics* statistics = opt->rep.statistics.get();
+  if (statistics) {
+    statistics->histogramData(type, data);
+  } else {
+    *data = rocksdb_statistics_histogram_data_t{};
```

Comment:
Not really, since data is a pointer to some existing memory. We probably can make the argument a pointer to pointer, but it would significantly complicate the API. Another option would be returning bool from the function. I can do that if you think it's better, don't have any hard feelings either way

Reply:
Oh right - I realized in c++ `data` is actually a constant pointer `HistogramData* const data`. Can we do the same here without any of the "We probably can make the argument a pointer to pointer, but it would significantly complicate the API. Another option would be returning bool from the function. I can do that if you think it's better, don't have any hard feelings either way" since the c++ one does not have it anyway.